atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraClientFactoryTest.java

```diff
@@ -44,22 +64,98 @@ public class CassandraClientFactoryTest {
                             .password("tom")
                             .build())
                     .usingSsl(false)
+                    .enableEndpointVerification(false)
                     .keyspace("ks")
                     .timeoutOnConnectionClose(Duration.ZERO)
                     .build());
 
+    private static final InetAddress DEFAULT_ADDRESS = mockInetAddress("1.2.3.4");
+
     private CassandraClient client = mock(CassandraClient.class);
     private PooledObject<CassandraClient> pooledClient = new DefaultPooledObject<>(client);
 
     @Test
-    public void reliesOnOpennessOfUnderlyingTransport() {
+    public void validateObject_reliesOnOpennessOfUnderlyingTransport() {
         when(client.getOutputProtocol()).thenReturn(new TCompactProtocol(new TMemoryInputTransport(), 31337, 131072));
```

Comment:
nit: thanks for making the test names uniform within the class. Just as a note, this was a rather legacy way of writing test names. We mostly use camel case these days.